dora/core/client/fs/src/main/java/alluxio/client/file/ufs/UfsBaseFileSystem.java

```diff
@@ -422,27 +421,18 @@ public void needsSync(AlluxioURI path) throws IOException, AlluxioException {
 
   @Override
   public Optional<String> submitJob(JobRequest jobRequest) {
-    try (CloseableResource<FileSystemMasterClient> client =
-        mFsContext.acquireMasterClientResource()) {
-      return client.get().submitJob(jobRequest);
-    }
+    throw new UnsupportedOperationException();
```

Comment:
Sometimes the error stack is not shown, so you'd struggle to find out what is exactly not supported.

```suggestion
    throw new UnsupportedOperationException("submitJob");
```